client/client-multi/client-objective-c/Kaa/BaseKaaClient.m

```diff
@@ -20,6 +20,10 @@
 
 @implementation BaseKaaClient
 
+- (instancetype)initWithPlatformContext:(id<KaaClientPlatformContext>)context delegate:(id<KaaClientStateDelegate>)delegate {
+    return [super initWithPlatformContext:context delegate:delegate autoGeneratedKeys:false];
```

Comment:
Also, I'd make the default strategy to generate keys for the user but not expect them to be already generated.